coldfront/core/allocation/views.py

```diff
@@ -104,12 +105,38 @@ def get_context_data(self, **kwargs):
         allocation_obj = get_object_or_404(Allocation, pk=pk)
         allocation_users = allocation_obj.allocationuser_set.exclude(
             status__name__in=['Removed']).order_by('user__username')
+        
+        def get_eula(alloc):
+            if alloc.get_resources_as_list:
+                for res in alloc.get_resources_as_list:
+                    if res.get_attribute(name='eula'):
+                        return res.get_attribute(name='eula')
+            else:
+                return None
+            
+        context['eulas'] = get_eula(allocation_obj)
+        context['res'] = allocation_obj.get_parent_resource.pk
+        context['res_obj'] = allocation_obj.get_parent_resource
 
         # set visible usage attributes
         alloc_attr_set = allocation_obj.get_attribute_set(self.request.user)
         attributes_with_usage = [a for a in alloc_attr_set if hasattr(a, 'allocationattributeusage')]
         attributes = alloc_attr_set
 
+
+        allocation_user_active_status_choice = AllocationUserStatusChoice.objects.get(
+                name='Active')
+        if EULA_AGREEMENT:
+            allocation_user_pending_status_choice = AllocationUserStatusChoice.objects.get(name='Pending')
+        
+        allocation_user_status = get_object_or_404(AllocationUser, allocation=allocation_obj, user=self.request.user).status
```

Comment:
This should check all users on an allocation and only show the ability to accept/deny EULA to the relevant user.  As is, this breaks views where the user is not a member of the allocation (e.g. an admin/staff view looking to approve/deny the allocation)